envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/routes/EnvoyEgressRoutesFactory.kt

```diff
@@ -359,7 +359,8 @@ class EnvoyEgressRoutesFactory(
                         .withClusterWeight(routeSpec.clusterName, routeSpec.clusterWeights.main)
                         .withClusterWeight(
                             getAggregateClusterName(routeSpec.clusterName, properties),
-                            routeSpec.clusterWeights.secondary
+                            routeSpec.clusterWeights.secondary,
+                            routeSpec.clusterName
```

Comment:
Do we need to pass any value to this header? How this header will be named? Maybe there should be a only boolean value which indicates if request was forwarded to secondary cluster

Reply:
so the header name is configured by this property headerName in SnpashotProperty, and the value I set as a cluster just so it can be possible to filter by it

Reply:
but it will be in <HeaderName>: <service-name> format?

Reply:
well I add it the same way as all other headers are added: `this.addResponseHeadersToAdd(buildHeader(key, value))` so I guess it will be in a correct format

Reply:
and I build HeaderValueOption which is envoy structure

Reply:
so what is a purpose of this header? to know if the request was splited to secondary cluster? if that is true, imo the value should be a boolean value or something different than service-name.

Reply:
and maybe for not splited traffic this values should be false

Reply:
kk I changed to boolean, but what would be a purpose of this tag with false value?

Reply:
hmm